packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -14,6 +14,10 @@
         ></span>
     </oui-message>
 
+    <pci-project-databases-warning
+        data-ng-if="$ctrl.showPaymentWarning([$ctrl.database])"
```

Comment:
array is passed as arguments : [$ctrl.database] 
in below comment, array is not passed. Is it ok? Can you keep consistency?